### PR TITLE
Add tox installation as prereq in contributing doc

### PR DESCRIPTION
**Pull request by jeremylinlin**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -6,13 +6,24 @@ The following is a set of best practices for contributing to the operator-courie
 This project uses [pipenv](https://github.com/pypa/pipenv) to manage dependencies and create virtual environments for development environment.
 
 ### Prerequisites
-Python3
```
Comment (kevinrizza) [inline docs/contributing.md]: Why add the space here? It just adds noise to the pr.

File: docs/contributing.md [hunk 2]
```diff
@@ -6,13 +6,24 @@ The following is a set of best practices for contributing to the operator-courie
...
+
+- tox
+
+  You could either install it globally
+  
+  - `sudo dnf install tox` 
+  
+  or install inside your pip environment (you will have to install it every time you enter the virtual environment)
+  
+  - `pip install tox`
```
Comment (kevinrizza) [inline docs/contributing.md]: Maybe we should just give one of these? There are a ton of ways to install things in lots of different environments (`apt-get`, `brew`, `yum` etc.). Can we just list the Fedora `dnf` option for simplification here? At least that is in line with the rest of the prereqs here.

Comment (jeremylinlin) [reply docs/contributing.md]: @kevinrizza Thanks Kevin. Simplified and updated.

**Pull request merged**